lib/pipeline.ts

```diff
@@ -6,6 +6,7 @@ import iam = require('@aws-cdk/aws-iam');
 import sns = require('@aws-cdk/aws-sns');
 import cdk = require('@aws-cdk/cdk');
 import path = require('path');
+import { PipelineWatcher } from './pipeline-watcher/watcher';
```

Comment:
Create an `pipeline-watcher/index.ts` file and `export * from './watcher'`, so here it will be:

```
import { PipelineWatcher } from './pipeline-watcher'
```